refinery/core/tests.py

```diff
@@ -788,6 +788,18 @@ def get_credentials(self):
             password=self.password
         )
 
+    def test_xml_format_ignored(self):
+        response = self.api_client.get(
+            '/api/v1/analysis/',
+            format='xml',
+            authentication=self.get_credentials()
+        )
+        self.assertValidJSONResponse(response)
+        self.assertEqual(
```

Comment:
I don't think this is necessary, or it should be a part of a different unit test.

Reply:
good call.